vnext/Microsoft.ReactNative/Fabric/Composition/ScrollViewComponentView.h

```diff
@@ -105,6 +105,11 @@ struct ScrollInteractionTrackerOwner : public winrt::implements<
       facebook::react::Tag tag,
       winrt::Microsoft::ReactNative::ReactContext const &reactContext);
 
+  void ScrollEnabled(bool status)
```

Comment:
noexcept - And maybe put the body in the cpp file.

Reply:
This method is not required now.